### DISCUSSION TITLE
The version 3.0

### DISCUSSION THREAD

**Discussion by ErrorCodesForLzx** [Melledy/Grasscutter#1626] [Q&A] [answered]
How can I get the gc server version 3.0?
Or say,can I play the genshin3.0?

**Comment by FeletexGee** [comment]
> How can I get the gc server version 3.0? Or say,can I play the genshin3.0?

I remembered there's a Grasscutter ver 1.2.2-dev- that supports Genshin version 3.0.But I can't find the link now.
*Hope it helps:-)*

**Reply by FeletexGee** [reply]
Oh,maybe [this jar file build from Actions](https://github.com/Grasscutters/Grasscutter/actions/runs/2811334964)may help.

**Reply by ErrorCodesForLzx** [reply]
Is there a way?can I modify the some config code  and recompile it?🧐

**Comment by KiminaNatsuki** [comment, marked as answer]
Please read the following answer from 4BenJ: https://github.com/Grasscutters/Grasscutter/discussions/1518#discussioncomment-3164595
